clients/kafka-common/src/main/java/org/eclipse/hono/kafka/client/tracing/KafkaTracingHelper.java

```diff
@@ -68,23 +67,23 @@ private KafkaTracingHelper() {
      * </ul>
      *
      * @param tracer The Tracer to use.
-     * @param topic The topic from which the operation name is derived.
+     * @param topic The topic to which the message is sent.
      * @param referenceType The type of reference towards the span context.
      * @param parent The span context to set as parent and to derive the sampling priority from (may be null).
      * @return The new span.
      * @throws NullPointerException if tracer or topic is {@code null}.
      */
-    public static Span newProducerSpan(final Tracer tracer, final HonoTopic topic, final String referenceType,
+    public static Span newProducerSpan(final Tracer tracer, final String topic, final String referenceType,
             final SpanContext parent) {
         Objects.requireNonNull(tracer);
         Objects.requireNonNull(topic);
         Objects.requireNonNull(referenceType);
 
-        return TracingHelper.buildSpan(tracer, parent, "To_" + topic.getType().endpoint, referenceType)
+        return TracingHelper.buildSpan(tracer, parent, "send message", referenceType)
```

Comment:
Could you use "kafka_send" (instead of "send message") as the operation name? This seems to be what the built-in tracing support of the Vert.x 4 Kafka client uses.

Reply:
@b-abel Do you see a practical advantage in using that pattern, being unlike the one used elsewhere in Hono?
FMPOV, having a common naming scheme across all spans created within Hono looks preferable.
In that sense, "send message" fits better IMHO.

Reply:
Vert.x 4.0 comes with [build-in](https://vertx.io/docs/vertx-kafka-client/java/#_automatic_trace_propagation) tracing support. I hope that we can use this and remove our custom tracing at least for the Kafka clients. Following the patterns used there could bring the advantage that the stored spans look the same instead of a sudden change.

Reply:
I think that remains to be seen, whether we should really be using the generic built-in tracing support with Vert.x 4 and abandon our Hono-specific tracing there (which still gives us more flexibility, I guess).
And if and when we do it, I think changing this one span name won't be a big deal.
Until then, I'd rather keep the span names consistent.